python-magic needs libmagic on Windows

I am trying to run beancount tests on Windows on AppVeyor. They fail because python-magic has some problem loading libmagic:
```
platform win32 -- Python 3.6.8, pytest-5.4.2, py-1.9.0, pluggy-0.13.1
rootdir: C:\projects\beancount
collected 1617 items / 10 errors / 1607 selected
=================================== ERRORS ====================================
_______________ ERROR collecting beancount/ingest/cache_test.py _______________
beancount\ingest\cache_test.py:9: in <module>
    from beancount.ingest import cache
beancount\ingest\cache.py:14: in <module>
    from beancount.utils import file_type
beancount\utils\file_type.py:22: in <module>
    import magic
C:\Python36\lib\site-packages\magic.py:179: in <module>
    libmagic = ctypes.CDLL(dll)
C:\Python36\lib\ctypes\__init__.py:348: in __init__
    self._handle = _dlopen(self._name, mode)
E   OSError: [WinError 193] %1 is not a valid Win32 application
```

I tries getting libmagic from here https://github.com/nscaife/file-windows, but I get the same error...

@yagebu How do you run fava tests on AppVeyor?

--- a/beancount/scripts/deps.py
+++ b/beancount/scripts/deps.py
@@ -125,7 +125,7 @@ def check_python_magic():
             # 'filemagic' is installed; install python-magic.
             raise ImportError
         return ('python-magic', 'OK', True)
-    except ImportError:
+    except (ImportError, OSError):
         return ('python-magic', None, False)
 
 

--- a/beancount/utils/file_type.py
+++ b/beancount/utils/file_type.py
@@ -26,7 +26,7 @@ try:
         warnings.warn("You have installed 'filemagic' instead of 'python-magic'; "
                       "disabling.")
         magic = None # pylint: disable=invalid-name
-except ImportError:
+except (ImportError, OSError):
     magic = None
 
 

